### DISCUSSION TITLE
No minified js version in the release

### DISCUSSION THREAD

**Discussion by ivangretsky** [alpinejs/alpine#3116] [3. Feature ideas]
Good day!

There is no minified ready to download js file in the release. Would be great to have it there, especially taking into account the target audience of the lib, that include a lot of "no-builders". 

Thanks!

**Comment by SimoTod** [comment]
The link in the documentation is actually the minified version.

**Reply by ivangretsky** [reply]
Are you talking about the CDN link? I was talking about downloadable link in [releases](https://github.com/alpinejs/alpine/releases). There are only source code downloads there now.

**Reply by SimoTod** [reply]
Yes, the no build version is served by a CDN and it's minified. Github only contains the source code and no compiled versions (either minified or not), it's basically the dev version.

**Reply by ivangretsky** [reply]
That's what I am talking about. I think the dist version should be in the releases to be easier acquired.

**Reply by SimoTod** [reply]
The problem is that if you commit a built version to a git repo, you get conflicts everytime you merge a pull request. It's just additional work for the maintainer. The no build version is on the CDN you can either link the CDN url or download the file if you prefer. I think (but I'm not sure) that the built version is also available via npm. Github is just for dev

**Reply by ivangretsky** [reply]
I am not talking about commiting it to the repo, but rather including in releases. Or should the dist files be in the repo to be included in the release...?

Anyway, Github is a perfect place to download stuff from) And it is not only for devs working on a project, but also for devs looking for code to work with in their projects. The mentioned workflow would make great sense for those including libs manually (like myself). And I would imagine those are some part of the alpine's target audience.

There is not a link for direct download even on alpinejs.dev.

**Reply by SimoTod** [reply]
I'm not sure if I follow you. A release in github is just a snapshot of what has been committed at that point in time. Sure, you can have a github action doing some additional work on tag but it's not how it woeks out of the box. 

Any javascript file can be downloaded if you paste the url in a new tab, right click and save (link on the documentation site).

Btw, I'm not the maintainer so it's not my call.

**Reply by ivangretsky** [reply]
Maybe we could ask @calebporzio ?
